ecosystem/web-wallet/package.json

```diff
@@ -31,8 +31,10 @@
     "start": "react-scripts start",
     "lint": "eslint ./src",
     "lint:fix": "eslint ./src --fix",
+    "tauri": "tauri",
```

Comment:
What does this do compared to the build:tauri?

Reply:
I can get rid of it if you don't think it's necessary @kent-white ?